internal/db/update/update.go

```diff
@@ -158,12 +158,29 @@ CREATE TABLE internal_cluster_members_new (
   UNIQUE(name),
   UNIQUE(certificate)
 );
-
-INSERT INTO internal_cluster_members_new SELECT id,name,address,certificate,1,(schema-1),heartbeat,role FROM internal_cluster_members;
+`
+		_, err = tx.ExecContext(ctx, stmt)
+		if err != nil {
+			return err
+		}
+
+		// Check if we have pre-emptively created this table when checking for schema updates. If so, we won't need to insert any rows.
+		stmt = `SELECT count(id) from internal_cluster_members_new`
+		err := tx.QueryRow(stmt).Scan(&count)
+		if err != nil {
+			return err
+		}
+
+		stmt = ""
+		if count == 0 {
+			stmt = "INSERT INTO internal_cluster_members_new SELECT id,name,address,certificate,1,(schema-1),heartbeat,role FROM internal_cluster_members;"
+		}
+
+		stmt = fmt.Sprintf(`%s
```

Comment:
Should this be in `if count == 0 { } else { "here?" }`. Otherwise the `stmt` statement in case `count == 0` will be overwritten.

Reply:
That's okay, we are appending the old `stmt` to the new `stmt`. If `count == 1`, `stmt` will be 2 lines. If `count == 0`, `stmt` will be 3 lines.

Reply:
Ah yes I see, missed the `fmt.Sprintf("%s ...", stmt)` part :)